mock/breez-sdk/src/lib.rs

```diff
@@ -1146,6 +1044,58 @@ fn generate_2_hashes_raw() -> (sha256::Hash, sha256::Hash) {
     (hash1, Hash::hash(hash1.as_byte_array()))
 }
 
+struct MockPayment {
+    payment_type: PaymentType,
+    amount_msat: u64,
+    fee_msat: u64,
+    description: Option<String>,
+    payment_hash: String,
+    payment_preimage: String,
+    destination_pubkey: String,
+    bolt11: String,
+    lnurl_pay_domain: Option<String>,
+    ln_address: Option<String>,
+    lnurl_metadata: Option<String>,
+    lnurl_withdraw_endpoint: Option<String>,
+    swap_info: Option<SwapInfo>,
+    reverse_swap_info: Option<ReverseSwapInfo>,
+}
+
+fn create_payment(p: MockPayment) -> Payment {
+    let now = Utc::now().timestamp();
+
+    Payment {
+        id: now.to_string(), // Placeholder. ID is probably never used
+        payment_type: p.payment_type,
+        payment_time: now,
+        amount_msat: p.amount_msat,
+        fee_msat: p.fee_msat,
+        status: PaymentStatus::Complete,
```

Comment:
Do we never need to create pending payments?

Reply:
Yeah maybe we do. Might be a future requirement or even bugfix.

However, for this refactoring I just checked what values are being used and the current code always uses `PaymentStatus::Complete` everyhwere, so that's why I went with that.